### PR TITLE
GSLUX-765: FIX drawing

### PR DESCRIPTION
**Pull request by AlitaBernachot**
### JIRA issue

https://jira.camptocamp.com/browse/GSLUX-765

### Description

Fix:
- style on button: was purple(why???) with exaggerated rounded corner too different from prod
- enable Escape keyup handler when drawing
- update coordinates for drawing modale when modifying drawn feature

### CODE REVIEW

File: src/services/utils.ts [hunk 1]
```diff
@@ -133,3 +133,7 @@ export function downloadFile(
   document.body.removeChild(a)
   URL.revokeObjectURL(url)
 }
+
+export function upperCaseFirst(str: string) {
```
Comment (mki-c2c) [inline src/services/utils.ts:137]: rename to "capitalize"

Comment (AlitaBernachot) [reply src/services/utils.ts]: done

**Comment by AlitaBernachot** [comment]
>This popup dragging is not present in V3.

Indeed, I have made some changes in the code to just update the modal anchor with a debounce, no more dragging effect.

> * automatically move the popup out of the way of the active part of the drawing
> * or enable manual dragging of the popup

Agree, it would have been nice to have the ability to move the modal but this is a whole new functionality, should be discussed first with client.

> I find it especially strange, when the popup moves in the opposite direction of a line vertex when creating a narrow v-shaped linestring.

This should be less visible as there is now a debounce when modifying the geom.

Is it better now?

**Comment by mki-c2c** [comment]
yes, it's better, nice. :+1: 

I don't see the point of moving the popup, but why not.

**Comment by AlitaBernachot** [comment]
> I don't see the point of moving the popup, but why not.

Why? => The center of the geom is supposed to be the modal anchor (it is how it is calculated).
You can move the geom far far far away from the modal. You won't see the info nor the profile updated. 

image

**Pull request merged**